### PR TITLE
added documentation for `blueprint` option

### PR DESCRIPTION
**Pull request by 62mkv**

### CODE REVIEW

File: pages/jdl/applications.md [hunk 1]
```diff
@@ -260,6 +260,12 @@ Here are the application options supported in the JDL:
     <td></td>
     <td></td>
   </tr>
+  <tr>
```
Comment (deepu105) [inline pages/jdl/applications.md:263]: @jhipster/developers didn't we change the option to blueprints? @murdos

Comment (62mkv) [reply pages/jdl/applications.md:263]: my understanding is that `blueprint` is still supported, but `blueprints` are not fully supported yet, see https://github.com/jhipster/jhipster-core/issues/415

**Comment by deepu105** [comment]
Ok thanks

Thanks & Regards,
Deepu

File: pages/jdl/applications.md (hunk 1 above)
Comment (MathieuAA) [reply pages/jdl/applications.md:263]: There's another one, `blueprint` is to be deprecated in favor of `blueprints`.

Comment (62mkv) [reply pages/jdl/applications.md:263]: so, should I add `blueprints` and specify that it's a list?

Comment (62mkv) [reply pages/jdl/applications.md:263]: (instead of `blueprint`)

Comment (MathieuAA) [reply pages/jdl/applications.md:263]: Ideally, both. The singular version is to be deprecated (but still valid as we speak), blueprints is valid.

**Comment by 62mkv** [comment]
added "blueprints" option, please look

File: pages/jdl/applications.md [hunk 2]
```diff
@@ -260,6 +260,18 @@ Here are the application options supported in the JDL:
...
+    <td>blueprint</td>
+    <td></td>
+    <td>name of an additional blueprint (see <a href="https://www.jhipster.tech/modules/marketplace/#/list">Marketplace</a>)</td>
+    <td>DEPRECATED. String</td>
+  </tr>
+  <tr>
+    <td>blueprints</td>
+    <td></td>
+    <td>names of additional blueprints (see <a href="https://www.jhipster.tech/modules/marketplace/#/list">Marketplace</a>)</td>
+    <td>Braces are mandatory</td>
```
Comment (MathieuAA) [inline pages/jdl/applications.md:273]: that's a syntax thing, no need to include it here

File: pages/jdl/applications.md [hunk 3]
```diff
@@ -260,6 +260,18 @@ Here are the application options supported in the JDL:
...
+  <tr>
+    <td>blueprint</td>
+    <td></td>
+    <td>name of an additional blueprint (see <a href="https://www.jhipster.tech/modules/marketplace/#/list">Marketplace</a>)</td>
+    <td>DEPRECATED. String</td>
+  </tr>
+  <tr>
+    <td>blueprints</td>
+    <td></td>
+    <td>names of additional blueprints (see <a href="https://www.jhipster.tech/modules/marketplace/#/list">Marketplace</a>)</td>
```
Comment (MathieuAA) [inline pages/jdl/applications.md:272]: Good idea to include the list!
Can you just capitalize "names", for consistency? thanks!

File: pages/jdl/applications.md [hunk 4]
```diff
@@ -260,6 +260,18 @@ Here are the application options supported in the JDL:
     <td></td>
     <td></td>
   </tr>
+  <tr>
+    <td>blueprint</td>
+    <td></td>
+    <td>name of an additional blueprint (see <a href="https://www.jhipster.tech/modules/marketplace/#/list">Marketplace</a>)</td>
```
Comment (MathieuAA) [inline pages/jdl/applications.md:266]: Good idea to include the list!
Can you just capitalize "name", for consistency? thanks!

**Comment by MathieuAA** [comment]
@62mkv that will be a great addition to the doc, I really like your idea to include the names of the blueprints!

**Comment by 62mkv** [comment]
by "include the names of the blueprints" you mean ... ? I originally put there something like 

```
  <tr>
    <td>blueprints</td>
    <td></td>
    <td>[vuejs,nodejs]</td>
    <td>Braces are mandatory. Names of additional blueprints (see <a href="https://www.jhipster.tech/modules/marketplace/#/list">Marketplace</a>)</td>
  </tr>
```

but then I realize that that column is otherwise used for "enumerated" where the provided options give the exhaustive list of possible values - mine would not be an example of such. Also, why would I promote just these 2 blueprints ? Authors of other blueprints could get offended. So I did not include that, after all

**Comment by MathieuAA** [comment]
You misunderstood, by saying you include the blueprint names, I mean by providing the link..

**Pull request merged**